BranchMonsterFactory/ViewControllers/MonsterViewerViewController.m

```diff
@@ -124,12 +124,13 @@ -(IBAction)shareSheet:(id)sender {
     commerceEvent.currency = @"USD";
     BNCProduct* branchester = [BNCProduct new];
     branchester.sku = [self.viewingMonster getMonsterName];
-    //branchester.name = [self.viewingMonster getMonsterName];
     commerceEvent.products = [NSArray arrayWithObject:branchester];
     
+    //[[Branch getInstance] userCompletedAction:BNCAddToCartEvent];
     [self.viewingMonster
      showShareSheetWithShareText:@"Share Your Monster!" completion:^(NSString * _Nullable activityType, BOOL completed) {
             if (completed) {
+                [[Branch getInstance] userCompletedAction:BNCAddToCartEvent];
```

Comment:
Can we move the 'Add to Cart' event to above the showing the share sheet?

Then the event fires even if the user cancels sharing.  It simulates someone adding something to a checkout cart but then deciding not to purchase.